pkg/chartverifier/profiles/profile.go

```diff
@@ -0,0 +1,100 @@
+package profiles
+
+import (
+	"errors"
+	"github.com/redhat-certification/chart-verifier/pkg/chartverifier/checks"
+	"gopkg.in/yaml.v3"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"runtime"
+)
+
+type Annotation string
+
+const (
+	DigestAnnotation                 Annotation = "Digest"
+	OCPVersionAnnotation             Annotation = "OCPVersion"
+	LastCertifiedTimestampAnnotation Annotation = "LastCertifiedTimestamp"
+)
+
+type Profile struct {
+	Apiversion  string       `json:"apiversion" yaml:"apiversion"`
+	Kind        string       `json:"kind" yaml:"kind"`
+	Name        string       `json:"name" yaml:"name"`
+	Annotations []Annotation `json:"annotations" yaml:"annotations"`
+	Checks      []*Check     `json:"checks" yaml:"checks"`
+}
+
+type Check struct {
+	Name string           `json:"name" yaml:"name"`
+	Type checks.CheckType `json:"type" yaml:"type"`
+}
+
+type FilteredRegistry map[checks.CheckName]checks.Check
+
+var profile *Profile
+
+func GetProfile() *Profile {
```

Comment:
since you are already in `profile` package, then the function name can be just `Get`, IMHO, it reads better like `profiles.Get()`

Reply:
Would like to defer this as the api will change in the next stage for multiple profiles and will be sure to remove the duplication.

Reply:
Done.